standardize_mutation_data.py

```diff
@@ -1381,6 +1383,64 @@ def map_data_values_to_header(header, line):
     data = dict(zip(header, list(map(str.strip, line.split("\t")))))
     return data
 
+def is_malformed_record(filename, record_index, line, header):
+    """
+        Verify whether the number of fields in the line matches the number of fields in the header.
+    """
+    split_line = list(map(str.strip, line.split("\t")))
+    if len(split_line) != len(header):
+        message = "[line %s] Encountered record in file where the number of fields in the line does not match the expected number of fields in the header, skipping processing of file..." % (record_index + 1)
+        print_warning(message)
+        PROBLEMATIC_FILES_REPORT[filename] = {"ERROR": [message]}
```

Comment:
because we set the value directly, won't this discard any warnings for the file encountered up to this point? Should we use a call to update_problematic_report_for_file() here instead?